src/cljs/bundled/lumo/build/api.cljs

```diff
@@ -37,15 +37,19 @@
 (defn build
   "Given a source which can be compiled, produce runnable JavaScript."
   ([source opts]
-   (build source opts
-     (env/default-compiler-env
-       (closure/add-externs-sources opts))))
+   (build source opts (fn [& args] args)))
   ([source opts compiler-env]
+   (build source opts compiler-env (fn [& args] args)))
+  ([source opts compiler-env cb]
```

Comment:
I was even wondering if we could accept the classic node `(fn [err data] ...)` callback so that you could `promisify` on it...but then I told myself: "KISS!!"